File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    eres = ""
    ores = ""
    for i in num_list:
        if i % 2 == 0: 
            eres += str(i)
        else:
            ores += str(i)
    answer = int(eres) + int(ores)
    return answer

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int ans = 0;
    int eres = 0;
    int ores = 0;
    for(int i=0; i<num_list.size(); i++) {
        if(num_list[i] % 2 == 0) {
            eres *= 10;
            eres += num_list[i];
        }
        else {
            ores *= 10;
            ores += num_list[i];
        }
    }
    
    ans = eres + ores;
    return ans;
}